ydb/core/tx/datashard/build_index.cpp

```diff
@@ -254,10 +255,9 @@ class TBuildScanUpload: public TActor<TBuildScanUpload<Activity>>, public NTable
 
         UploadStatusToMessage(progress->Record);
 
+        LOG_N("Finish" << Debug() << " " << progress->Record.ShortDebugString());
```

Comment:
Maybe also `LOG_I`?

Reply:
Default log level is Notice, and I expect to at least see start/finish logs as these scan operation is quite expensive in database scope